controllers/ghost.js

```diff
@@ -4,24 +4,35 @@ const request = require('request-promise');
 
 const router = express.Router();
 
+let cache = null;
+let lastUpdatedTimestamp = 0;
+
 router.get('/:slug', async (req, res, next) => {
 	if (!Configuration.has('GHOST_API_KEY') || !Configuration.has('GHOST_API_URL')) {
 		return next(new Error('GHOST_API_URL or/and GHOST_API_KEY are not defined'));
 	}
 
-	const options = {
-		uri: `${Configuration.get('GHOST_API_URL')}/content/pages/slug/${req.params.slug}/`,
-		qs: {
-			key: Configuration.get('GHOST_API_KEY'),
-			fields: 'html,title',
-		},
-		json: true,
-		timeout: Configuration.get('REQUEST_TIMEOUT_MS'),
-	};
+	if (lastUpdatedTimestamp < Date.now() - 1000 * 60 * 5) {
```

Comment:
&& cache !== null
```suggestion
	if (cache == null || lastUpdatedTimestamp < Date.now() - 1000 * 60 * 5) {
```
Create something like GHOST_CACHE_TIME_SECONDS